src/app/views/sidebar/resource-explorer/collection/EditScopePanel.tsx

```diff
@@ -1,10 +1,11 @@
-import { DefaultButton,
-  DialogFooter,
+import * as React from 'react';
```

Comment:
QQ, why do some files have this import and then individual imports of hooks etc below it?

Reply:
Hmm, checking project wide, seems to be only this file with multiple React imports. Any other instance noted?